client/views/admin/AdminHistoryView.vue

```diff
@@ -62,14 +74,38 @@
     
     methods: {
       load() {
-        this.$http.get('/totalOodlet', {
+        if(this.selectedOffice === 'TOTALS') {
+          this.isTotal = true;
```

Comment:
Couldnt do this: 
```
let params = {
          fromDate: moment(this.fromDate).format(),
          toDate: moment(this.toDate).format(),
          office: this.selectedOffice
        }
```
Because api doesnt accept redundant param so made it ternary like this:
```
let options = this.selectedOffice === 'TOTALS' ?
           {
            params: {
              fromDate: moment(this.fromDate).format(),
                toDate: moment(this.toDate).format()
            }
          } :
          {
            params: {
              fromDate: moment(this.fromDate).format(),
              toDate: moment(this.toDate).format(),
              office: this.selectedOffice
            }
          };
```